server/cli/src/refresh_dates.rs

```diff
@@ -364,6 +364,7 @@ mod tests {
                 r.item_id = mock_item_a().id;
                 r.store_id = mock_store_a().id;
                 r.expiry_date = Some(NaiveDate::from_ymd(2023, 02, 01));
+                r.name_id = Some(String::from("name_store_b"));
```

Comment:
technically not needed for this test and inline_init allows for it to not be specified (keeps test data specific to test case)